README.md

```diff
@@ -263,3 +335,7 @@ conditions.
     <img src="https://raw.githubusercontent.com/astral-sh/uv/main/assets/svg/Astral.svg" alt="Made by Astral">
   </a>
 </div>
+
+[commands]: https://github.com/astral-sh/ty/blob/main/docs/cli.md
```

Comment:
I switched to using full URls because pypi doesn't support relative links

Reply:
We should probably restore the PyPI readme transform script so these links work properly on branches in PRs, but this seems fine to start

Reply:
Doesn't it just replace badges/images?

Reply:
It looks like in uv the relative link is turned into a tagged link? (from looking at PyPI)

Reply:
that's cool!